doc/nrf/config_and_build/configuring_app/sysbuild_images.rst

```diff
@@ -3,11 +3,19 @@
 Sysbuild images
```

Comment:
Can you update https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/config_and_build/config_and_build_system.rst#sysbuild with information about these new pages about sysbuild (this one, https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst , and https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/config_and_build/configuring_app/sysbuild_forced_options.rst )?